tcms/rpc/api/testrun.py

```diff
@@ -238,6 +238,8 @@ def filter(query=None):  # pylint: disable=redefined-builtin
             "product_version",
             "start_date",
             "stop_date",
+            "planned_start",
+            "planned_stop",
```

Comment:
Commit log needs a message to indicate that there are new fields returned by the API. All of the 3 APIs in fact. See other commit log messages in the branch or in earlier history.